### ISSUE TITLE
List effects after effect available in keyframes 

### ISSUE THREAD

**Issue by sinzen** [facebookincubator/Keyframes#140]
Hi, I just begin with keyframes,and I want to know the effects available to export in keyframes library.. There are now item listed to that ? Thanks

**Comment by jhughes696** [comment]
Yes! I'm also new to Keyframes and I've found it pretty difficult to find a good list of supported features
